tests/robot-tests/tests/admin/bau/comments.robot

```diff
@@ -200,3 +200,6 @@ Switch to bau1 to approve release for immediate publication
     user waits until parent contains element    ${details}    xpath:.//a[text()="Edit this release"]
     user clicks link    Edit this release
     user approves original release for immediate publication
+
+Visit methodology on public side and check it is visible on 'all methodologies' page
+    user verifies methodology is visible on public methodology page    %{TEST_THEME_NAME}    %{TEST_TOPIC_NAME}    ${PUBLICATION_NAME}    /methodology/ui-tests-comments-%{RUN_IDENTIFIER}
```

Comment:
I'm not sure why this comments test really needs to have anything to do with setting up a methodology at all? Would be interested to know.

Reply:
Realistically we could get rid of the methodology assertions in this test. I put the methodology assertions in this test to mirror the type of content analysts create on production. Happy to remove or discuss more if you feel it's not necessary (would reduce test time if we removed it)